cdec/installation-manager-cli-bundle/src/main/java/com/codenvy/im/cli/command/LoginCommand.java

```diff
@@ -32,46 +33,38 @@
     @Argument(name = "password", description = "The user's password", required = false, multiValued = false, index = 1)
     private String password;
 
-    @Option(name = "--remote", description = "Name of the remote codenvy", required = false)
-    private String remoteName;
+    @Option(name = "--remote", description = "Codenvy on-prem URL", required = false)
```

Comment:
Since it isn't a remote name anymore, I would prefer to change option name to `codenvyUrl`